Update guide with enonic dev command #10 (#11)
enonic/doc-starter-headless: docs/index.adoc

== Building and Deploying

> To build and start the app, run this command. *Go with the default options*:

Reviewer: ```suggestion
To build and start the app, run this command (*go with the default options* when asked for sandbox details):
```

  enonic dev

[NOTE]
====
`enonic dev` command will start your sandbox in detached dev mode, and also execute the `dev` gradle task on the hmdb starter, providing you a smooth experience while developing with Enonic. For further details on Enonic CLI's dev mode, read the https://developer.enonic.com/docs/enonic-cli/stable/dev#dev[docs].
====

[NOTE]
====

> To verify that your app started successfully, you may find an entre like this within in the sandbox log:

Reviewer: ```suggestion
If your app started successfully, there will be an entry similar to this one in the sandbox log:
```
